integration-tests/src/test/resources/model-in-image/model.cluster.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+# A sparce model file to add a Configured Cluster 
+
+topology:
+    Cluster:
+        'cluster-2':
+    Server:
+        'config-managed-server-1':
+            Cluster: 'cluster-2'
+            ListenPort: '@@PROP:MANSVR_PORT@@'
+            NumOfRetriesBeforeMsiMode: 0
```

Comment:
Are the MSI properties required?  I want to make sure that this isn't a workaround to a bug...

Reply:
No this is not a  workaround. I had a sample model

Reply:
Then please remove the attributes.

Reply:
Removed in my latest commit